dev-tools/reconfigurator-cli/tests/output/cmds-set-remove-mupdate-override-stdout

```diff
@@ -0,0 +1,324 @@
+using provided RNG seed: reconfigurator-cli-test
+> # Load example system with 7 sleds:
+> #
+> # sled 0: unset -> unset (unchanged)
+> # sled 1: unset -> set
+> # sled 2: set -> unset
+> # sled 3: set -> set (unchanged)
+> # sled 4: set -> set (changed)
+> # sled 5: set -> set (unchanged) but change something else
+> # sled 6: set -> sled removed
+> #
+> # We'll also add another sled below (new_sled_id) with
+> # remove_mupdate_override set.
+> #
+> # We don't need any zones for this test, so disable that to keep the
+> # outputs minimal.
```

Comment:
https://github.com/oxidecomputer/omicron/pull/8142 makes this much easier to read.